examples/okta_domain/basic.tf

```diff
@@ -1,3 +1,4 @@
 resource "okta_domain" "test" {
-  name   = "testAcc-replace_with_uuid.example.com"
```

Comment:
The endpoint no longer allow domain to be named testAcc-randomnumber.example.com